csharp/ql/src/semmle/code/csharp/dataflow/SSA.qll

```diff
@@ -19,7 +19,7 @@ module Ssa {
       }
 
       /**
-       * Holds if the this field or any of the fields part of the qualifier
+       * Holds if this field or any of the fields part of the qualifier
```

Comment:
Does the `qualifier` part still apply? This `this.getQualifier().(FieldOrPropSourceVariable).isVolatile()` was removed from the predicate.

Reply:
Good catch, I'll fix